site/_components/case-studies.njk

```diff
@@ -0,0 +1,31 @@
+<!--{#
+  Renders a grid of case study cards.
+
+  @param {object} content: An object containing the following attributes.
+  @param {string} content.title: Section title, displayed in a heading element.
+  @param {string} content.subtitle (optional): Section subtitle, displayed beneath the heading.
+  @param {string} content.modifiers (optional): Modifier classes to be applied to the section.
+
+  @param {object} content.cta (optional): A CTA object containing the following required attributes.
+  @param {string} content.cta.text: Text to be displayed within the CTA.
+  @param {string} content.cta.url: href value for the CTA.
+  @param {object[]} studies: Array of case studies.
+
+  @param {string} locale: Code for the current locale, required when content.cta is present.
+#}-->
+{% from 'card-study.njk' import cardStudy %} {% macro caseStudies(content, studies, locale) %}
+<section class="case-studies {% if content.modifiers %}{{content.modifiers}}{% endif %}">
+  <header class="case-studies__header">
+    <h1 class="type--h1 case-studies__title">{{content.title}}</h1>
+    {% if content.subtitle %}
+    <h2 class="type--large case-studies__subtitle">{{content.subtitle}}</h2>
+    {% endif %}
+  </header>
+  <div class="case-studies__content">{% for study in studies %}{{cardStudy(study, loop.first)}}{% endfor %}</div>
+  {% if content.cta %} {% set localeCta = { 'url': '/' ~ locale ~ '/' ~ content.cta.url, 'text': content.cta.text }%}
```

Comment:
I have other comments about how to handle this in other PRs; let's align on one (maybe as a separate PR) and update all accordingly